domain/feed/src/main/java/com/into/websoso/feed/GetFeedsUseCase.kt

```
@@ -0,0 +1,4 @@
+package com.into.websoso.feed
+
+class GetFeedsUseCase {
+}
```

detekt (1.23.8)

[warning] The class or object GetFeedsUseCase is empty.

(detekt.empty-blocks.EmptyClassBlock)